project-base/storefront/components/Blocks/Product/LastVisitedProducts/LastVisitedHelpers.ts

```diff
@@ -15,7 +15,10 @@ export const getLastVisitedProductCatalogNumbers = (context?: GetServerSideProps
 
 export const setLastVisitedProductCatalogNumbers = (newLastVisitedProductsCatalogNumber: string): void => {
     const currentLastVisitedProductsCatalogNumbers = getLastVisitedProductCatalogNumbers() || [];
-    currentLastVisitedProductsCatalogNumbers.unshift(newLastVisitedProductsCatalogNumber);
+
+    if (currentLastVisitedProductsCatalogNumbers.indexOf(newLastVisitedProductsCatalogNumber) === -1) {
```

Comment:
What about 

`if (currentLastVisitedProductsCatalogNumbers.includes(newLastVisitedProductsCatalogNumber)) {
        currentLastVisitedProductsCatalogNumbers.unshift(newLastVisitedProductsCatalogNumber);
    }`

I would consider it as better readable.

Reply:
Yes it is little bit better. Fixed.